crypto/crypto_test.cc

```diff
@@ -73,6 +73,24 @@ TEST(CryptoTest, Strndup) {
   EXPECT_STREQ("", str.get());
 }
 
+TEST(CryptoTest, OPENSSL_hexstr2buf) {
+  const char *test_cases[][2] = {{"a2", "\xa2"},
+                                 {"a213", "\xa2\x13"},
+                                 {"ffeedd", "\xff\xee\xdd"},
+                                 {"10aab1c2", "\x10\xaa\xb1\xc2"}};
+
+  for (auto test_case : test_cases) {
+    const char *test_value = test_case[0];
+    const char *expected_answer = test_case[1];
+    size_t actual_answer_len = 0;
+    size_t expected_answer_len = OPENSSL_strnlen(test_case[1], 5);
```

Comment:
Is the length `5` too short here?

Reply:
The longest test case I have only uses 4 bytes, so there's currently no need to go larger unless we add longer test cases.